pyogrio/tests/test_raw_io.py

```diff
@@ -906,6 +906,25 @@ def test_read_datetime_millisecond(test_datetime):
     assert field[1] == np.datetime64("2020-01-01 10:00:00.000")
 
 
+@pytest.mark.parametrize("prefix_driver", [True, False])
+def test_read_unsupported_ext(tmp_path, prefix_driver):
```

Comment:
I'd suggest splitting this into 2 test cases, since the logic is a bit different:

`test_read_unsupported_ext` (the error case)
`test_read_unsupported_ext_with_prefix`

Reply:
Sure, I first wrote it like that, but merged them to reuse the test data and make it clearer that with the same data it works with prefixing versus without.

But it doesn't really matter...